### PR TITLE
feat: improve Discord interaction UX

### PR DESCRIPTION
**Pull request by nexpid**
when using the bot, I noticed that the string select interaction disappeared *wayy* too quickly, since I didn't know what response each intent represented.  
this PR fixes that by giving the user 10 more seconds, and also adds a short preview of the reply that intent is associated with! :3

### CODE REVIEW

**Review by castdrian — APPROVED** [review]
looks good, will test later

File: src/utils/autosupport.ts [hunk 1]
```diff
@@ -40,39 +40,47 @@ export async function getResponse(message: Message) {
...
 
-			if (config.devGuildId) {
-				for (const [, guildResponses] of responseCache) {
-					if (guildResponses) {
-						for (const [key, value] of guildResponses) {
-							aggregatedResponses.set(key, value);
-						}
-					}
-				}
+export function getResponseContent(intentName: string, guild: string) {
```
Comment (castdrian) [inline src/utils/autosupport.ts:67]:
```suggestion
export function getResponseContent(intentName: string, guildId: string) {
```

File: src/utils/autosupport.ts [hunk 2]
```diff
@@ -40,39 +40,47 @@ export async function getResponse(message: Message) {
...
-			});
 		}
 
-	} catch (error) {
-		message.client.logger.error(error);
-		return undefined;
+		return aggregatedResponses.get(intentName)!;
+	}
+
+	const guildResponses = responseCache.get(guild);
```
Comment (castdrian) [inline src/utils/autosupport.ts:82]:
```suggestion
	const guildResponses = responseCache.get(guildId);
```

**Review by castdrian — CHANGES_REQUESTED** [review]
other than the param name this is good to go

**Comment by nexpid** [comment]
goat

**Pull request merged**